Astral Pathfinder/Models/Planet.hpp

```diff
@@ -0,0 +1,32 @@
+//
+//  Planet.hpp
+//  Astral Pathfinder
+//
+//  Created by Ian Holdeman on 2/10/18.
+//  Copyright © 2018 Ian Holdeman. All rights reserved.
+//
+
+#ifndef Planet_hpp
+#define Planet_hpp
+
+// MARK: Libraries and Frameworks
+#include "SDL2/SDL.h"
+
+class Planet {
+public:
+  Planet();
+  ~Planet();
+  
+  SDL_Point position;
+  
+  // resources
+  int deposits, fertility;
```

Comment:
this should be made private with getter function so they are made read only outside this class, eventually we may have a mineDeposits() or some such method that will update the deposits internally